tsconfig.json

```diff
@@ -2,37 +2,27 @@
   "compilerOptions": {
     "allowJs": true,
     "esModuleInterop": true,
+    "emitDeclarationOnly": true,
     "allowSyntheticDefaultImports": true,
     "jsx": "react-jsx",
     "noEmit": false,
     "noImplicitAny": false,
     "forceConsistentCasingInFileNames": true,
     "strict": true,
-    "typeRoots": ["./node_modules/@types"],
     "sourceMap": true,
     "inlineSources": true,
     "resolveJsonModule": true,
     "skipLibCheck": true,
-    "module": "ESNext",
+    "module": "Preserve",
```

Comment:
**note:** Because we're using a bundler, we need to use `module: Preserve` to ensure TypeScript deals with module imports in the same way as modern bundlers do (e.g. no need for a file extension)